stories/topics/ref-aap-gcp-automation-performance.adoc

```diff
@@ -2,7 +2,7 @@
 
 = Automation performance
 
-Automating against the managed node allotment in the offer is provided with the following operational expectations. 
+Automating against your licensed managed node allotment in the offer is provided with the following operational expectations. 
```

Comment:
```suggestion
Automating against your licensed managed node allocation in the offer is provided with the following operational expectations. 
```
Would it be accurate to use "allocation"? Translation programs may find this less ambiguous.